modules/setting/setting.go

```diff
@@ -749,6 +749,11 @@ please consider changing to GITEA_CUSTOM`)
 		"StampMicro":  time.StampMicro,
 		"StampNano":   time.StampNano,
 	}[Cfg.Section("time").Key("FORMAT").MustString("RFC1123")]
+	// Determine When using custom time format like '2006-01-02 15:04:05'
+	if (TimeFormat == "") {
```

Comment:
Ok forget my comments for now --> that's the problem which prevent me from using custom time format ..
in my `conf/app.ini`
    [time]
    FORMAT = 2006-01-02 15:04:05
this FORMAT is not in the map ... so `TIMEFORMAT` will be empty ..
this code will save `2006-01-02 15:04:05` in `TIMEFORMAT` 
so when using function like TimeSince will return 
    <span class="time-since poping up" title="" data-content="2017-01-30 11:56:19" data-variation="inverted tiny">5 ساعات مضت</span>
instead of 
    <span class="time-since poping up" title="" data-content="" data-variation="inverted tiny">5 ساعات مضت</span>